txnsync/interfaces.go

```diff
@@ -71,7 +71,10 @@ type NodeConnector interface {
 	Random(uint64) uint64
 	GetPeers() []PeerInfo
 	GetPeer(interface{}) PeerInfo // get a single peer given a network peer opaque interface
-	UpdatePeers([]*Peer, []interface{})
+	// UpdatePeers call is being made to inform the node that either a link need to be established
+	// between the set of the txsyncPeers peers and the set of netPeers, or that the averageDataExchangeRate
+	// was recalculated and could potentially be updated.
+	UpdatePeers(txsyncPeers []*Peer, netPeers []interface{}, averageDataExchangeRate uint64)
```

Comment:
Could you mention the units of the rate somewhere?

Also, it isn't clear that this is the `averagePerPeerDataExchangeRate`

Reply:
yes. I'll rename & clarify documentation.